creator-node/src/serviceRegistry.js

```diff
@@ -26,27 +26,26 @@ class ServiceRegistry {
     this.ipfsLatest = ipfsLatest
     this.blacklistManager = BlacklistManager
     this.monitoringQueue = new MonitoringQueue()
+    this.nodeConfig = config
```

Comment:
moving forward we want to try and pull more services off serviceRegistry, so mde this change for new components to consume

Reply:
i agree with this sentiment. not sure if configs should be treated the same but not opposed

Reply:
cool